server/xpub-controller/helpers/files.js

```diff
@@ -24,7 +24,7 @@ class FilesHelper {
       manuscriptId,
       url: `manuscripts/${manuscriptId}`,
       filename,
-      type: 'MANUSCRIPT_SOURCE_PENDING',
+      type: 'MANUSCRIPT_SOURCE_PENDING', // TODO figure out what this is for
```

Comment:
To differentiate between an already existing `MANUSCRIPT_SOURCE` file in the db and one that is pending being uploaded. The removal of the old file doesn't happen till later in the upload function so this is to prevent the db ever having two `MANUSCRIPT_SOURCE` files for one manuscript.